package.json

```diff
@@ -1,10 +1,11 @@
 {
   "private": true,
-  "version": "2019-03",
+  "version": "2019-04",
```

Comment:
while we don't publish this to npm I think that we should stick with the [semver](https://docs.npmjs.com/misc/semver) syntax for the version field. eg.: 2019.4.1 or 2019.4.0